packages/shared-components/src/models/dropdown/dropdown-item.ts

```diff
@@ -15,13 +15,18 @@ export class DropdownItem<T> {
   /**
    * The dropdown item tooltip. It will be used if present; otherwise, the {@link DropdownItem._tooltipLabelKey} will be used.
    */
-  private _tooltip: string = '';
+  private _tooltip: string | (() => Promise<string>);
```

Comment:
here again we have 3 instances of this return type! I think we can just have something like `export type Awaitable = () => Promise<T>` and use it instead.

Reply:
Done